app/controllers/api/v1/forms_controller.rb

```diff
@@ -1,11 +1,11 @@
 class Api::V1::FormsController < ApplicationController
   def index
     org = params[:org]
-    forms = if org.present?
-              Form.where(org:)
-            else
-              Form
-            end
+    creator_id = params[:creator_id]
+
+    forms = Form.where(nil)
```

Comment:
I don't understand this 😅 Is this the same as `Form.all`?

Reply:
Yes, yes it is! I'm using that instead now 🙃